src/Samples/dotnet6server/Program.cs

```diff
@@ -1,24 +1,42 @@
 using CoreWCF;
 using CoreWCF.Configuration;
+using CoreWCF.Channels;
+using CoreWCF.Description;
 using dotnet6Server;
 using MyContracts;
 
+const string BASE_HTTP_URL = "http://localhost:5000";
```

Comment:
what about running on Azure? Is it possible to avoid hardcoded urls and use a startdard way of asp.net? At the moment I'm having my project running locally but it does not start on azure. I suspect because I don't set urls

Reply:
The BaseURL is a bit funky. The host:port is only used for the generation of wsdl, specifying the address of the endpoint. The path is critical as that specifies a base address that is prepended to the paths as part of the ServiceEndpoint registration.
The problem is that at this part of the process activation, ASP.NET hasn't figured out what its listening on yet, so we can't ask it, or I'd have put that in the sample.

That's all why this PR is not yet merged.

What URLs have you told kestrel to listen on? Do you have a Urls specified in app.config or via a command line option?